### ISSUE TITLE
RFE: Config option to hide scores

### ISSUE THREAD

**Issue by kevinclevenger** [tonycpsu/mlbstreamer#37]
Really need this, I often don't have time to watch the game until a day or two after it's played.

TIA!

**Comment by tonycpsu** [comment]
Well, the good news is I added support for this a while ago.  The bad news is I forgot I'd created an issue for it.  :)

In your `config.yaml` file, use the hide_spoiler_teams option with a list of teams you'd like to hide scores for, e.g.:

```
        hide_spoiler_teams:
        - PHI
```

It'd probably be nice to have an "all" option for people who never want to see any scores, so I'll leave this issue open to remind me to implement that.

**Comment by kevinclevenger** [comment]
At the moment it's only 8 teams, so not too bad :-) "All" would be really welcome for next season.

Question: If I only have LAD listed as hidden will that also hide (in this case) MIL?

And also a *huge* thanks for mlbstreamer, and picking up where mlbviewer left off.

**Comment by tonycpsu** [comment]
Yeah, it hides the score of any game that team is involved -- away or home.
